ts/IterableInboxMessageList.tsx

```diff
@@ -1,31 +1,25 @@
-'use strict';
-import React, { ReactElement } from 'react';
-import { ScrollView, Text, StyleSheet } from 'react-native';
-import IterableInboxMessageCell from './IterableInboxMessageCell';
+'use strict'
+import React, { ReactElement } from 'react'
+import { ScrollView, Text, StyleSheet } from 'react-native'
+import IterableInboxClickableRow from './IterableInboxClickableRow'
 
 type MessageListProps = {
    messages: Array<any>,
    handleMessageSelect: Function  
 }
 
 const IterableInboxMessageList = ({ messages, handleMessageSelect }: MessageListProps) => {
-   function flaggedMessage(message: { [key: string]: any }, index: number) {
-      return (index === messages.length - 1) ?
-         <IterableInboxMessageCell
-            index={index} 
-            message={message}
-            handleMessageSelect={(index: number) => handleMessageSelect(index, messages)} 
-            last={true} /> :
-         <IterableInboxMessageCell
-            index={index} 
-            message={message}
-            handleMessageSelect={(index: number) => handleMessageSelect(index, messages)}  
-            last={false} />
-   }
 
    function displayMessages() {
       return messages.map((message, index) => {
-         return flaggedMessage(message, index)
+         let last = (index === messages.length - 1) ? true : false
```

Comment:
you don't need the `true : false` thing here.  `index === messages.length - 1` is either always `true` or `false`. 

`const isLast = index === messages.length - 1` will do.